client/operation_join.go

```diff
@@ -24,5 +24,7 @@ func (op operationJoinResponse) Process(state *albionState, uploader iuploader)
 	}
 
 	state.LocationId = loc
+	state.CharacterName = op.CharacterName
```

Comment:
Can you add a debug line mentioning we are setting the character name, like the location one below.

Reply:
Can you check to see if this operation has the character ID as well as the character name? I would like that to get sent with private uploads as well as the name.

Reply:
Sadly, it does not seem to have this information. I'm currently trying to look to the events to see if the ID can be found somewhere. I don't really get their ID system anyway... For example, I cannot seem to retrieve my id even using their API: https://gameinfo.albiononline.com/api/gameinfo/search?q=Oxisto yields nothing, although my character exists :) Searching for other characters work

Reply:
The character IDs are UUIDs, you can definitely see them when you look at the market JSON string.

Names are unique so it should be good enough for now, I was just really hoping that join response would contain the character ID. The request might though. If you feel like looking into it feel free, but I wont hold up this PR for it.